src/tests/ConfigureRecordingSetTests.test.tsx

```diff
@@ -0,0 +1,176 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable jest/prefer-expect-assertions */
+
+import { cleanup, fireEvent, render, screen, waitFor } from '@testing-library/react';
+import i18next from 'i18next';
+import React from 'react';
+import { initReactI18next } from 'react-i18next/';
+
+import ConfigureRecordingSetPage from '../renderer/components/configure-recording-set-page/index';
+import { RecordingSet } from '../renderer/types';
+
+const flushPromises = () => new Promise(setImmediate);
```

Comment:
What is this for?

Reply:
flushPromises forces the test to wait (with a timeout) to insure that all the promises that were created in a call return before the test moves on.  I use it though out the tests to make sure that everything actually finishes updating before i check for the changes

Reply:
OK, in that case, maybe put this in its own `test-utils.ts`